src/components/tickets/ticket/create-ticket/create-ticket.tsx

```diff
@@ -60,170 +64,170 @@ export class CreateNewTicket extends React.Component<
 
-                <div>
-                  <p>
-                    <strong>Set reminder:</strong>
-                  </p>
-                  <Checkbox
-                    toggle
-                    label="Include a reminder"
-                    checked={this.state.setReminder}
-                    onChange={() => {
-                      const flippedState = !this.state.setReminder
-                      this.setState({ setReminder: flippedState })
-                      if (this.state.setReminder !== true) {
-                        const [date, time] = formatDateTime(new Date())
-                        this.setState({
-                          remindDate: date,
-                          remindTime: time,
-                          remindMessage: this.state.description,
-                        })
-                      } else {
-                        this.setState({
-                          remindDate: null,
-                          remindTime: null,
-                          remindMessage: null,
-                        })
-                      }
-                    }}
-                  />
-                </div>
-                {this.state.setReminder ? (
-                  <div>
-                    <Divider />
-                    <Label htmlFor={'remindMessage'}>Message:</Label>
-                    <br />
-                    <TextArea
-                      row={4}
-                      col={20}
-                      name="remindMessage"
-                      placeholder="Give a short remind message (max 100 characters)"
-                      value={this.state.remindMessage}
-                      onChange={this.handleChange}
-                      maxLength={100}
-                    />
-                    <br />
-                    <DateTimePicker
-                      handleChange={this.handleChange}
-                      datepicker={{
-                        name: 'remindDate',
-                        value: this.state.remindDate,
-                      }}
-                      timepicker={{
-                        name: 'remindTime',
-                        value: this.state.remindTime,
-                      }}
-                    />
-                  </div>
-                ) : null}
-                <br />
-                <Divider />
-                <Button type="submit" disabled={!this.validityCheck()}>
-                  Create
-                </Button>
-              </Form>
+                          createNewTicket({
+                            variables: {
+                              ticket: {
+                                assignedTo: this.state.assignedTo,
+                                type: this.state.type,
+                                remindNotificationDate: this.state.remindDate,
+                                remindNotificationTime: this.state.remindTime,
+                                remindMessage: this.state.description,
```

Comment:
We should probably not include a reminder if it was not toggled to be included

```suggestion
                                remindNotificationDate: this.state.setReminder ? this.state.remindDate : null,
                                remindNotificationTime: this.state.setReminder ? this.state.remindTime : null,
                                remindMessage: this.state.setReminder ? this.state.description : null,
```

Reply:
Also, test to ensure that no reminder is sent if "setreminder" is not toggled

Reply:
Tested